src/app/components/story-navigation-bar/story-navigation-bar.component.html

```diff
@@ -1,64 +1,82 @@
 <section>
   <header [attr.aria-busy]="!storylist">
-    <ng-container *ngIf="storylist && storylist.title; else titleSkeleton">
-      <a
-        [routerLink]="'/' + appRouteTree['STORYLIST']"
-        [queryParams]="{ slug: storylist.slug }"
-      >
-        <h3 class="storylist-title">{{ storylist.title }}</h3>
-      </a>
-    </ng-container>
+    @if(!!storylist && storylist.title){
+    <a
+      [routerLink]="'/' + appRouteTree['STORYLIST']"
+      [queryParams]="{ slug: storylist.slug }"
+    >
+      <h3 class="storylist-title">{{ storylist.title }}</h3>
+    </a>
+    } @else {
+    <ng-container *ngTemplateOutlet="titleSkeleton"></ng-container>
+    }
   </header>
 
-  <ng-container *ngIf="!!storylist; else bodySkeleton">
-    <ng-container *ngFor="let publication of displayedPublications">
-      <a
-        [routerLink]="
-          publication.published ? ['/' + appRouteTree['STORY']] : null
+  @if(!!storylist){ @for(publication of displayedPublications; track $index){
+  <a
+    [routerLink]="publication.published ? ['/' + appRouteTree['STORY']] : null"
+    [queryParams]="{ slug: publication.story.slug, list: storylist.slug }"
+  >
+    <article
+      [ngClass]="{ selected: selectedStorySlug === publication.story.slug }"
+    >
+      <cuentoneta-story-edition-date-label
+        class="edition-and-label"
+        [label]="
+          publication.published
+            ? getEditionLabel(publication, publication.publishingOrder)
+            : storylist.comingNextLabel
         "
-        [queryParams]="{ slug: publication.story.slug, list: storylist.slug }"
-      >
-        <article [ngClass]="{ 'selected': selectedStorySlug === publication.story.slug }">
-          <cuentoneta-story-edition-date-label
-            class="edition-and-label"
-            [label]="
-              publication.published
-                ? getEditionLabel(publication, publication.publishingOrder)
-                : storylist.comingNextLabel
-            "
+      />
+      @if(publication.published){
```

Comment:
Excelente lo de los if, else y el for, así sacamos el mayor rendimiento de Angular 🎉